porcupine/plugins/filemanager.py

```diff
@@ -316,6 +316,23 @@ def open_in_file_manager(path: Path) -> None:
     subprocess.Popen([opener_command, str(path)])
 
 
+def open_in_terminal(path: Path) -> None:
+    # Using pushd for windows to change drive aswell navigate to directory
+    windowingsystem = get_main_window().tk.call("tk", "windowingsystem")
+    if windowingsystem == "win32":
+        command = "pushd"
+    else:
+        command = "cd"
+
+    subprocess.Popen(f"start /wait {command} {path}", shell=True)
```

Comment:
This has a couple problems:
- I don't think it works if the path has spaces in it. For example `C:\Foo Bar` would turn into `start /wait pushd C:\Foo Bar`, which looks wrong to me.
- It doesn't work at all on macos and linux. There's no `start` command on these operating systems.

Starting a terminal is trickier than you might expect, and the `run` plugin already has code that deals with it. We should reuse it somehow, without copy/pasting anything of course, and I think that will be tricky.

You can just delete `open_in_terminal()` from this PR. You can then work on it in a separate PR, or just wait for me to eventually do it.

Reply:
I did test paths with spaces, there wasn't a problem on windows 11 at least.
I didn't consider the "start" command only working on windows, my bad. That does make it a bit trickier, but I'll look in to the run plugin code and see if I'm able to reuse it. Thanks!